README.md

```diff
@@ -90,8 +67,14 @@ int main()
         /* Retry if DNS resolution query failed. */
         if( dnsStatus != 0 )
         {
-            /* Get back-off value (in milliseconds) for the next retry. */
-            retryStatus = BackoffAlgorithm_GetNextBackoff( &retryParams, &nextRetryBackOff );
+            /* Generate a random number and get back-off value (in milliseconds) for the next retry.
+             * Note: It is recommended to either use a True Random Number Generator (TRNG) for
+             * calculation of unique back-off values in devices so that collision between
+             * devices attempting retries at the same intervals can be avoided.
+             *
+             * For the simplicity of the code example, the pseudo random number generator, rand() function
+             * is used. */
+            retryStatus = BackoffAlgorithm_GetNextBackoff( &retryParams, rand(), &nextRetryBackOff );
```

Comment:
Where is this PRNG seeded? I ask because if you do an OTA on a fleet they will all start up with the same seed and you will have exactly the connection storm this is intended to avoid.

Reply:
Good catch. I have added seeding logic of RNG in the example.